meta-efi-secure-boot/recipes-bsp/grub/grub-efi_2.02.bbappend

```diff
@@ -35,15 +35,20 @@ CFLAGS_append = " -fno-toplevel-reorder"
 inherit user-key-store
 
 python __anonymous () {
+    import re
+
+    target = d.getVar('TARGET_ARCH', True)
+
+    # functions efi_call_foo and efi_shim_exit are not implemented for arm64 yet
+    if re.match('aarch64', target):
+        raise bb.parse.SkipPackage("grub-efi is incompatible with target %s" % target)
+
```

Comment:
It looks like you wanna disable grub-efi build if meta-secure-boot is configured but disabled. So why not use COMPATIBLE_HOST?

Reply:
Do you mean set COMPATIBLE_HOST for aarch64?

COMPATIBLE_HOST_aarch64 = 'null'

If so, I'll send a new PR. Thanks.